packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -305,6 +305,13 @@ export function getOdspResolvedUrl(resolvedUrl: IResolvedUrl): IOdspResolvedUrl
 	return resolvedUrl as IOdspResolvedUrl;
 }
 
+/**
+ * @public
+ */
+export function isOdspResolvedUrl(resolvedUrl: IResolvedUrl): resolvedUrl is IOdspResolvedUrl {
+	return "odspResolvedUrl" in resolvedUrl;
```

Comment:
```suggestion
	return "odspResolvedUrl" in resolvedUrl && resolvedUrl.odspResolvedUrl===true;
```

Reply:
can you also change the implementation of getOdspResolvedUrl to use this new function